templates/manifest.json

```diff
@@ -96,7 +112,15 @@
                 "build": "latest",
                 "memoryMbytes": 1024,
                 "timeoutSecs": 3600
-            }
+            },
+            "showcaseFiles": [
+                "src/main.py",
+                "src/__main__.py",
+                "src/items.py",
+                "src/pipelines.py",
+                "src/settings.py",
+                "src/spiders/title_spider.py"
```

Comment:
Here I would only show `src/main.py`, `src/pipelines.py` and `src/spiders/title_spider.py`, the rest are not that important.

Reply:
same concern as above comment

Reply:
In that case, i keep __main__.py as well to unify it with the rest of python scrapers.